### PR TITLE
docs(DatePicker): refactor jsx examples to tsx

### PR DESCRIPTION
**Pull request by langz**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/components/date-picker/Examples.tsx [hunk 1]
```diff
@@ -260,3 +261,60 @@ export const DatePickerScreenshotTests = () => {
     </Wrapper>
   )
 }
+
+export const DatePickerDateFns = () =>
+  global.IS_TEST ? null : (
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/components/date-picker/Examples.tsx:266]: We may consider to use `globalThis.IS_TEST`. Does it work for you too?

**Review by tujoworker — APPROVED** [review]

**Comment by tujoworker** [comment]
Nice work 🔥

File: packages/dnb-design-system-portal/src/docs/uilib/components/date-picker/Examples.tsx (hunk 1 above)
Comment (langz) [reply packages/dnb-design-system-portal/src/docs/uilib/components/date-picker/Examples.tsx:266]: I'll change this in a new PR 🚀

**Pull request merged**